src/Akeneo/Platform/Job/back/Infrastructure/Query/SearchJobExecution.php

```diff
@@ -71,9 +83,27 @@ public function count(SearchJobExecutionQuery $query): int
     SELECT count(*)
     FROM akeneo_batch_job_execution je
     JOIN akeneo_batch_job_instance ji on je.job_instance_id = ji.id
+    %s
 SQL;
+        $whereSqlPart = '';
+
+        $type = $query->type;
+
+        if (!empty($type)) {
+            $whereSqlPart = 'WHERE ji.type IN (:type)';
+        }
+
+        $sql = sprintf($sql, $whereSqlPart);
```

Comment:
As we are going to accumulate filters

```suggestion
        $whereParts = [];
        $type = $query->type;

        if (!empty($type)) {
            $whereParts[] = 'ji.type IN (:type)';
        }

        $sql = sprintf($sql, empty($whereSqlPart) ? '' : 'WHERE ' . implode($whereSqlPart, ' AND '));
```